sql/core/src/main/scala/org/apache/spark/sql/DataFrameWriter.scala

```diff
@@ -573,6 +575,21 @@ final class DataFrameWriter[T] private[sql](ds: Dataset[T]) {
     format("csv").save(path)
   }
 
+  private def runCommand(session: SparkSession, name: String)(command: LogicalPlan): Unit = {
```

Comment:
the problem is that, in some commands, like `InsertIntoHiveTable`, we already use `SQLExecution.newExecution`, so we can't use it again to wrap these commands.

In the future we should figure out a central place to put `SQLExecution.newExecution`